src/compiler/scala/tools/nsc/typechecker/RefChecks.scala

```diff
@@ -21,30 +21,22 @@ import scala.tools.nsc.settings.NoScalaVersion
 import symtab.Flags._
 import transform.Transform
 
-/** <p>
- *    Post-attribution checking and transformation.
- *  </p>
- *  <p>
- *    This phase performs the following checks.
- *  </p>
- *  <ul>
- *    <li>All overrides conform to rules.</li>
- *    <li>All type arguments conform to bounds.</li>
- *    <li>All type variable uses conform to variance annotations.</li>
- *    <li>No forward reference to a term symbol extends beyond a value definition.</li>
- *  </ul>
- *  <p>
- *    It performs the following transformations.
- *  </p>
- *  <ul>
- *   <li>Local modules are replaced by variables and classes</li>
- *   <li>Calls to case factory methods are replaced by new's.</li>
- *   <li>Eliminate branches in a conditional if the condition is a constant</li>
- *  </ul>
+/** Post-attribution checking and transformation.
  *
- *  @author  Martin Odersky
+ *  This phase performs the following checks:
```

Comment:
```suggestion
 *  This phase checks the following postconditions:
```

Reply:
Alternatively, `performs the following song and dance`.